package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "swagger-router",
-  "version": "0.5.6",
+  "version": "0.6.0",
```

Comment:
There is only one change since 0.5.6 and this PR doesn't carry backwards-incompatible changes, so perhaps increasing just the patch portion would be better?